src/Bubble.tsx

```diff
@@ -319,6 +320,17 @@ export default class Bubble<
     return null
   }
 
+  renderFooter() {
+    const {
+      currentMessage,
+      renderFooter
+    } = this.props
+    if (renderFooter && currentMessage) {
```

Comment:
what do you think if do this:

```
renderFooter() {
    const {
      currentMessage,
      renderFooter
    } = this.props

   if (renderFooter && currentMessage) return renderFooter(currentMessage)
   return null
}
```

Reply:
No logical difference and my version is using gifted chat coding style.